airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/datatypes.sql

```diff
@@ -59,6 +59,10 @@
     String
 {%- endmacro -%}
 
+{#-- TODO: Remove this macro when dbt issue regarding unlimited varchars on postgres is resolved (https://github.com/airbytehq/airbyte/issues/12775) --#}
```

Comment:
doublechecking - is this meant to link https://github.com/dbt-labs/dbt-core/issues/5238 ?

To my understanding, this PR closes https://github.com/airbytehq/airbyte/issues/12775 (but once the dbt-core issue is closed, we should upgrade to the new dbt-core/dbt-utils version?)

Reply:
nice, makes sense! let's clarify that in the comment and then :shipit:

```suggestion
{#-- TODO: Remove this macro when dbt issue regarding unlimited varchars on postgres is resolved (https://github.com/dbt-labs/dbt-core/issues/5238) and we've upgraded to the latest version of dbt --#}
```